pom.xml

```diff
@@ -81,6 +81,7 @@
         <module>spring</module>
         <module>proto</module>
         <module>tests</module>
+        <module>dozer-spring-boot-auto-configuration</module>
```

Comment:
can this be renamed to: spring-boot

Reply:
I would call it `dozer-spring-boot` just to have clear relation between module folder name and module name. 

Actually, I would do the same with `dozer-spring` module. Having `dozer-spring` module in `spring` folder a little bit confusing :)

Reply:
i'd suggest to change the structure as suggested by @orange-buffalo to another PR, and keep this focused on boot.